pkgs/by-name/bo/boring/package.nix

```
@@ -0,0 +1,44 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "boring";
+  version = "0.11.4";
+
+  src = fetchFromGitHub {
+    owner = "alebeck";
+    repo = "boring";
+    rev = version;
```

```suggestion
    tag = version;
```

suggestion: Preferring tag makes robust src.